yt_dlp/extractor/pixivsketch.py

```diff
@@ -0,0 +1,118 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import (
+    traverse_obj,
+    ExtractorError,
+    unified_timestamp,
+)
+
+
+class PixivSketchBaseIE(InfoExtractor):
+    def _call_api(self, video_id, path, referer):
+        response = self._download_json(f'https://sketch.pixiv.net/api/{path}', video_id, headers={
```

Comment:
This can be modified like this to better be inline with the vars and it's working 
```suggestion
        response = self._download_json(f'https://sketch.pixiv.net/api/{path}{video_id}.json', video_id, headers={
```

Reply:
doing so makes reading code harder ig

Reply:
i disagree tbh.
If you see the usage of this and how all other extractors work, path is strictly just the path/endpoint and the fact that you are passing id separately as well as in the path looks weird.

Reply:
actually how i see it is, you are sacrificing the readability in the declaration but improving the readability in the calls by a lot

Reply:
if it's going to investigate why user call can't resolve to ongoing live, joining `video_id` makes everything break
see https://github.com/ytdl-patched/ytdl-patched/blob/ab35b1b7c15c68ea9b76f62c6212b49c65579346/yt_dlp/extractor/pixivsketch.py#L94 (very old code before rewrite)